projects/ngx-location-viewer/src/lib/components/layer-management/layer-management.component.scss

```diff
@@ -0,0 +1,16 @@
+h1 {
+    font-size: 1.44rem !important;
+}
+
+h2 {
+    font-size: 1.2rem !important;
+}
```

Comment:
Our core branding has utility heading classes to solve these problems. In this case `.h5` and `.h6`.